src/transform/xsl/default.xsl

```diff
@@ -1422,8 +1422,8 @@
             <div class="acta-fig-image-caption-wrapper">
                 <div class="fig-expansion">
                     <div class="fig-inline-img">
-                        <a href="{@xlink:href}" class="figure-expand-popup" title="{$caption}">
-      <img data-img="{$graphics}" src="{@xlink:href}" class="responsive-img">
+                        <a href="{@xlink:href}" class="figure-expand-popup" title="{$caption}" data-lightbox="article-figures" data-title="{$caption}">
+      <img data-img="{$graphics}" src="{@xlink:href}" class="responsive-img img-fluid">
```

Comment:
I quite like this, can we also add the alt-text if present? (`data-alt` on lightbox)

You might need a rebase to get 8c6fbaddedca54fa429ae4fab662ba97e601fb20 in your branch